File: hw1/src/main/java/Calculator.java
public class Calculator {

    public static void calculate(int val1, String operator, int val2) {
        int result = 0;
        if (operator.equals("/") && val2 == 0) System.out.print("Сan't divide by 0");
        else {
            switch (operator) {
                case "+" -> result = val1 + val2;
                case "-" -> result = val1 - val2;
                case "*" -> result = val1 * val2;
                case "/" -> result = val1 / val2;
            }

            System.out.print("Result is: " + result);
        }
    }

    public static boolean isInt(String arg) {
        try {
            Integer.parseInt(arg);
            return true;
        } catch (NumberFormatException e) {
            return false;
        }
    }
}

File: hw5/src/main/kotlin/Calculator.kt
class Calculator {
    fun calculate(val1: Int, operator: String, val2: Int) {
        if (operator == "/" && val2 == 0) print("Сan't divide by 0")
        else {
            val result = when (operator) {
                "+" -> val1 + val2
                "-" -> val1 - val2
                "*" -> val1 * val2
                "/" -> val1 / val2
                else -> null
            }
            if (result == null) print("Incorrect operator")
            else print("Result is: $result")
        }
    }

    fun isInt(arg: String): Boolean {
        return try {
            Integer.parseInt(arg)
            true
        } catch (e: NumberFormatException) {
            false
        }
    }
}